packages/runtime/example/box/index.html

```diff
@@ -40,14 +40,13 @@
                   properties: {
                     events: [
                       {
-                        event: 'click',
+                        event: 'onClick',
```

Comment:
I think the `onEvent` naming may not be very friendly to editor users. For example, when they configure events handler in the GUI, `click`, `submit` looks more straightforward.